### PR TITLE
Lock for Android docs v2 updates

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/libraries/lock-android/index.md [hunk 1]
```diff
@@ -1,478 +1,248 @@
...
-  <% } %>
-<% }); %>
-</ul>
+Get started using Lock for Android below, or if you're looking for a specific document, try the listing of [additional documents](#additional-documents) related to Lock for Android!!
 
 ## Requirements
 
 Android API level 15+ is required in order to use Lock's UI.
-If you'll create your own API and just call Auth0 API via the `com.auth0.android:core:1.13.+`, the minimum required API level is 9.
+If you intend to create your own API and just call Auth0 API via the `com.auth0.android:auth0:1.1.0`, the minimum required API level is also 15+.
```
Comment (hzalaz) [inline articles/libraries/lock-android/index.md]: `create your own UI` instead of `create your own API`

Comment (hzalaz) [inline articles/libraries/lock-android/index.md]: Also, let's mention https://github.com/auth0/Auth0.Android lib instead (we should probably aim to have a custom UI guide)

File: articles/libraries/lock-android/index.md [hunk 2]
```diff
@@ -1,478 +1,248 @@
...
-</activity>
-<meta-data android:name="com.auth0.lock.client-id" android:value="@string/auth0_client_id"/>
-<meta-data android:name="com.auth0.lock.domain-url" android:value="@string/auth0_domain"/>
-<!--Auth0 Lock End-->
+compile 'com.auth0.android:lock:2.0.0'
 ```
 
-> The value `@string/auth0_client_id` is your application's clientID and `@string/auth0_domain` is your tenant's domain in Auth0, both values can be found in your app's settings.
-> The final value of `android:scheme` must be in lowercase
+_You can check for the latest version on the repository [Releases](https://github.com/auth0/Lock.Android/releases) tab or in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22lock%22%20g%3A%22com.auth0.android%22)_
```
Comment (hzalaz) [inline articles/libraries/lock-android/index.md]: Let's add JCenter too, in Android is more common to use it since it's the default repository for gradle

**Review by hzalaz — CHANGES_REQUESTED** [review]

**Review by hzalaz — APPROVED** [review]

**Pull request merged**